spec/facets/ErrorMessageRunFacet.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema" : "https://json-schema.org/draft/2020-12/schema",
+  "$id" : "https://openlineage.io/spec/facets/1-0-0/ErrorMessageRunFacet.json",
+  "$defs" : {
+    "ErrorMessageRunFacet" : {
+      "allOf" : [ {
+        "$ref" : "https://openlineage.io/spec/1-0-2/OpenLineage.json#/$defs/RunFacet"
+      }, {
+        "type" : "object",
+        "properties" : {
+          "message" : {
+            "description" : "A human-readable string representing error message generated by observed system",
+            "type" : "string",
+            "example" : "org.apache.spark.sql.AnalysisException: Table or view not found: wrong_table_name; line 1 pos 14"
+          },
+          "programmingLanguage" : {
+            "description" : "Programming language the observed system uses.",
+            "type" : "string",
+            "example" : "JAVA"
```

Comment:
I would decide on lowercase or uppercase, without having explicit enum as there are 100s of programming languages.

Having optional version is a good idea. We can "steal" more ideas later: https://github.com/OpenLineage/OpenLineage/issues/446#issuecomment-1006811072

Reply:
BTW, kinda philosophical question: maybe `programmingLanguage` and `languageVersion` should belong to separate facet? It still could be attached only when job fails. Do we want more "composable" facets or every possible information available in one?

Reply:
Yeah, separate facet would be a better match. Perhaps this could be something like `platform`, so that it could include frameworks like Spark or Flink. This can be done in separate PR.

Reply:
We know framework/integration from `producer` field right now.